types/signed_header.go

```diff
@@ -103,31 +116,37 @@ func (sh *SignedHeader) ValidateBasic() error {
 		return err
 	}
 
-	// Handle Based Rollup case
-	if sh.Validators == nil || len(sh.Validators.Validators) == 0 {
-		return nil
-	}
-
 	if err := sh.Validators.ValidateBasic(); err != nil {
 		return err
 	}
 
+	// Rollkit vA enforces a centralized sequencer.
+	if sh.Validators == nil || len(sh.Validators.Validators) != 1 {
```

Comment:
`sh.Validators.ValidateBasic()` already checks for `nil|empty`, so you can just do `len(sh.Validators.Validators) != 1`